src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -24,14 +24,15 @@ const TableHeadRow = styled.tr`
   vertical-align: center;
   height: 2rem;
 `
-const TableBodyRow = styled.tr`
+const TableBodyRow = styled.tr<{ pending?: boolean }>`
   &:nth-child(even) {
     background-color: ${({ theme }) => theme.core.darkBackground};
   }
   &:hover {
     background-color: ${({ theme }) => theme.core.alternateBackground};
   }
   height: 2rem;
+  ${({ pending }) => pending && "background-color: #5e626d"}
```

Comment:
It looks good on dark-mode, but it doesn't look good on light-mode. The best way to ensure that it looks good on both modes is to use one of the colors defined under `theme.core`.